### PR TITLE
Add command to publish Storybook to personal fork

### PR DESCRIPTION
**Pull request by lazytype**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Create and document workflow to publish storybook to personal fork of leafygreen-ui](https://jira.mongodb.org/browse/PD-771)

## 🛠 Types of changes

- [X] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,12 @@ You can deploy a static build of our Storybook site to gh-pages from the master
 1. First be sure you've built a static version of Storybook: `yarn build:storybook`
 2. Then deploy to gh-pages: `yarn release:site`
 
+### To deploy to your own mirror of leafygreen-ui
```
Comment (hswolff) [inline README.md:158]: Does this assume you've already created your fork?

File: scripts/publish-storybook.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
```
Comment (hswolff) [inline scripts/publish-storybook.sh:1]: This is fantastic! In some ways should be its own npm package. We could create a new repo on https://github.com/mongodb-js and publish on there if we want. May add more overhead than we want right now.

File: README.md (hunk 1 above)
Comment (lazytype) [reply README.md:158]: Oh, yeah it does. I need to think about how to approach that case.

File: scripts/publish-storybook.sh (hunk 2 above)
Comment (lazytype) [reply scripts/publish-storybook.sh:1]: Probably more maintainable to rewrite in JS if it's not gonna be just a one-off and would also allow us to import the gh-pages package directly. One thing I wanted to do was make the storybook build happen it the background while you're prompted for username and such, but couldn't get ctrl+c to kill the child process. I imagine there must be something in npm that could make that easier.

**Review by DesignerDave — APPROVED** [review]
I don't know why this wasn't approved before. Was anything blocking this? LGTM

**Pull request merged**